### PR TITLE
Review project

### PR DESCRIPTION
**Pull request by Shaheryar0054**
### Review my own project

- I copied my previews code
- setup linters
- Added Readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (Shaheryar0054) [inline index.html:149]: Please do not leave unused code in your codebase. Unused code as comments makes your code not clean.

**Review by Shaheryar0054 — COMMENTED** [review]
Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
**Highlights!**

- Followed Gitflow👍
- Professional readme👍
- Descriptive pull request👍
 # Status: Required changes:♻

- Please check my comments below and make all required changes.
- There are some indentation errors in your styel.css you also need to solve that errors.
- Kindly add a **.gitignore** file in your project-2 branch 👍 because it is a very important file for all your projects 😊
- After adding it, it would be nice if you add **node_modules** at the very first line of the **.gitignore file** 🤔 to ignore external dependencies which will just increase the size of 


Submit for re-review when you are done.

Thanks 😊

**Review by Shaheryar0054 — COMMENTED** [review]
Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
**Highlights!**
- Followed Gitflow👍
- Professional readme👍
- Descriptive pull request👍
 # Status: Required changes:♻
- Please check my comments below and make all required changes.
- There are some indentation errors in your styel.css you also need to solve that errors.
- Kindly add a **.gitignore** file in your project-2 branch 👍 because it is a very important file for all your projects 😊
- After adding it, it would be nice if you add **node_modules** at the very first line of the **.gitignore file** 🤔 to ignore external dependencies which will just increase the size of 
 


Submit for re-review when you are done.

Thanks 😊

**Review by Olamarx — APPROVED** [review]
Hi @Shaheryar0054, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Grammatically correct comments ✅ 
- Good use of the GitHub review feature 👍🏼 
- Used inline comments ✔️


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**